profiler/src/ProfilerEngine/Datadog.Profiler.Native/PInvoke.cpp

```diff
@@ -109,4 +87,36 @@ extern "C" void __stdcall SetApplicationInfoForAppDomain(const char* runtimeId,
         serviceName ? serviceName : std::string(),
         environment ? environment : std::string(),
         version ? version : std::string());
+}
+
+extern "C" void __stdcall SetEndpointForTrace(const char* runtimeId, uint64_t traceId, const char* endpoint)
+{
+    if (!CorProfilerCallback::GetClrLifetime()->IsRunning())
```

Comment:
This test should be done AFTER GetInstance() != nullptr test is done (GetClrLifetime() does not check if _this/GetInstance() is not null)

Reply:
Good point, but then we have the same issue in all others p/invokes

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/3015/commits/fec54e15782805f7e9565c90f54f5135668fc49d